learn/wavemaker-release-notes/v10-5-6.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "WaveMaker 10.5.6 - Release date: 08 October 2020"
+id: ""
+sidebar_label: "Release v10.5.6"
+---
+*WaveMaker released a new update version with bug fixes.*
+
+## Improvements
+
+---
+
+- Stackable multiple toaster messages. This offers a way to display multiple toasters where you can configure the time duration.
+- Update of Android **targetSdkVersion** is required by November 2, 2020. For more information, see [Mobile Build - Manual](learn/hybrid-mobile/mobile-build-manual).
+
+## Bug Fixes
+
+---
+
+- Fixed issue related to exporting project shell to EDN.
+- Fixed issues related to editing and displaying the user profile name.
+- Fixed issue related to default messages which are not localized.
+- Fixed issue related to the **onSuccess** handler for file upload variable to customize the upload success notification.
```

Comment:
Should be changed to "When onSuccess handler for a service variable used to upload file is overridden, the default success toaster is now suppressed. This allows users to customise the behaviour as per need."